mods/tuxemon/l18n/es_ES/LC_MESSAGES/base.po

```diff
@@ -1950,10 +1982,15 @@ msgstr "<recovery_module>: GREETINGS ${{name}}. PLEASE UNDERSTAND THIS AREA NEVE
 "<recovery_module>: WAKE UP. FORGET ALL. ALL IS BUT A DREAM..."
 
 msgid "37707_villager_male_1"
-msgstr "Blank: Mmmm. Hola. Lo siento: normalmente somos precavidos con los extraños. Solo recibimos visitas... a veces.\n"
-"Blank: Pero ya que estás aquí, quizás puedas hacerme un favor. Puede sonar extraño viniendo de mí, pero necesito saberlo...\n"
-"Blank: ¿Sabes quién soy? ¿Me has... visto alguna vez? Tengo curiosidad: mi nombre, ¿tengo un...\n"
-"Blank: No. Lo siento: he cometido un error. Por favor, nunca hables de esto. Continúa, disfruta tu viaje aquí."
+msgstr ""
+"Blank: Mmmm. Hola. Lo siento: normalmente somos precavidos con los extraños. "
+"Solo recibimos visitas... a veces.\n"
+"Blank: Pero ya que estás aquí, quizás puedas hacerme un favor. Puede sonar "
+"extraño viniendo de mí, pero necesito saberlo...\n"
+"Blank: ¿Sabes quién soy? ¿Me has... visto alguna vez? Tengo curiosidad: mi "
+"nombre, ¿tengo un...\n"
+"Blank: No. Lo siento: he cometido un error. Por favor, no le diga a ninguno "
+"que le pregunté. Continúa, disfruta tu viaje aquí."
```

Comment:
I think the older translation, although not literally the same as the original, is more idiomatic. In either case, "usted" is rarely used in peninsular Spanish, especially with young people:

```suggestion
"Blank: No. Lo siento: he cometido un error. Por favor, no le digas a nadie "
"que te he preguntado eso. Continúa, disfruta tu viaje aquí."
```